csrc/id_model/indexing.cpp

```diff
@@ -1022,4 +1007,59 @@ void TensorIndexer::setupAllocationDomains(const std::vector<Expr*>& exprs) {
   alloc_info_ = std::move(alloc_setup.tv_alloc_info_map);
 }
 
+std::vector<PredicateInfo> TensorIndexer::getInlinePredicates(
+    TensorView* tv,
+    const Expr* expr,
+    const std::vector<ForLoop*>& for_loops) const {
+  const auto& zero_val = tv->fusion()->zeroVal();
+
+  const std::vector<IterDomain*>& predicate_domains =
+      getPredicateDomains(tv, expr);
+
+  const IndexingInfo& index_info = computeIndex(
+      expr, traversalGraph().toGroups(predicate_domains), for_loops);
+
+  const auto& index_map = index_info.index_map;
+
+  std::vector<PredicateInfo> info_vec;
+  info_vec.reserve(predicate_domains.size());
+  std::unordered_set<ValGroup> already_indexed_domains;
```

Comment:
Is this unused?

Reply:
Ah, that'll be eventually used but not in this PR yet. Just forgot to remove it from #2238 when I stripped out except for the bare minimum code. Removed.